website/docs/index.html.markdown

```diff
@@ -1,108 +1,103 @@
 
-* `credentials` - (Optional) Contents of a file that contains your service
-  account private key in JSON format. You can download your existing
-  [Google Cloud service account file]
-  from the Google Cloud Console, or you can create a new one from the same page.
+The Google provider is jointly maintained by:
 
-  Credentials can also be specified using any of the following environment
-  variables (listed in order of precedence):
+* The [Google Cloud Graphite Team](https://cloudplatform.googleblog.com/2017/03/partnering-on-open-source-Google-and-HashiCorp-engineers-on-managing-GCP-infrastructure.html) at Google
+* The Terraform team at [HashiCorp](https://www.hashicorp.com/)
 
-    * `GOOGLE_CREDENTIALS`
-    * `GOOGLE_CLOUD_KEYFILE_JSON`
-    * `GCLOUD_KEYFILE_JSON`
+If you have configuration questions, or general questions about using the provider, try checking out:
 
-  The [`GOOGLE_APPLICATION_CREDENTIALS`][adc]
-  environment variable can also contain the path of a file to obtain credentials
-  from.
+* The [Google Cloud Platform Community Slack](https://gcp-slack.appspot.com/) #terraform channel
+* [Terraform's community resources](https://www.terraform.io/docs/extend/community/index.html)
+* [HashiCorp support](https://support.hashicorp.com) for Terraform Enterprise customers
 
-  If no credentials are specified, the provider will fall back to using the
-  [Google Application Default Credentials][adc].
-  If you are running Terraform from a GCE instance, see [Creating and Enabling
-  Service Accounts for Instances][gce-service-account] for details.
+## Features and Bug Requests
 
-  On your computer, if you have made your identity available as the
-  Application Default Credentials by running [`gcloud auth application-default
-  login`][gcloud adc], the provider will use your identity.
+The Google provider's bugs and feature requests can be found in the [GitHub repo issues](https://github.com/terraform-providers/terraform-provider-google/issues).
+Please avoid "me too" or "+1" comments. Instead, use a thumbs up [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)
+on enhancement requests. Provider maintainers will often prioritise work based on the
+number of thumbs on an issue.
 
-  ~> **Warning:** The gcloud method is not guaranteed to work for all APIs, and
-  [service accounts] or [GCE metadata] should be used if possible.
+Community input is appreciated on outstanding issues! We love to hear what use
+cases you have for new features, and want to provide the best possible
+experience for you using the Google provider.
 
-* `project` - (Optional) The ID of the project to apply any resources to.  This
-  can also be specified using any of the following environment variables (listed
-  in order of precedence):
+If you can't find an existing issue for a bug or feature request you have
```

Comment:
I would break this into 2 different sentences, this doesn't seem to flow well right now as a bulleted list.

Reply:
Reworded